uber/config.py

```diff
@@ -273,7 +271,6 @@ def CURRENT_ADMIN(self):
     def HTTP_METHOD(self):
         return cherrypy.request.method
 
-    @property
     def SUPPORTER_COUNT(self):
```

Comment:
i feel like even though we can't use the decorator form here, we should probably mark these with a comment like:
```
# IMPORTANT - Cherrypy Request-specific caching in place here, see request_cached_property()
```